backend/src/main/java/gov/cdc/usds/simplereport/api/patient/PatientResolver.java

```diff
@@ -26,6 +32,15 @@ public long patientsCount(UUID facilityId, boolean showDeleted, String namePrefi
     return ps.getPatientsCount(facilityId, showDeleted, namePrefixMatch);
   }
 
+  public boolean patientExists(
+      String firstName, String lastName, LocalDate birthDate, String zipCode, UUID facilityId) {
+    Organization org = os.getCurrentOrganization();
+    Optional<Facility> facility =
+        facilityId == null ? Optional.empty() : Optional.of(os.getFacilityInCurrentOrg(facilityId));
+
+    return ps.isDuplicatePatient(firstName, lastName, birthDate, zipCode, org, facility);
```

Comment:
when `facilityId` is `null`, can we still locate a patient?

Reply:
Yes: a `null` facility ID indicates that a user is attempting to register a patient with all facilities in an organization.